packages/server/modules/user-emails/migrations/20240703084247_user-emails.ts

```diff
@@ -0,0 +1,30 @@
+import { Knex } from 'knex'
+import { USER_EMAILS_TABLE_NAME } from '../constants'
+
+export async function up(knex: Knex): Promise<void> {
+  await knex.schema.createTable(USER_EMAILS_TABLE_NAME, (table) => {
```

Comment:
@alemagio You generally don't want to use imported constants in migrations, cause in case the constant changes the migration is gonna run differently (which should never happen)